packages/react-component-library/src/components/TabBase/partials/StyledTab.tsx

```diff
@@ -52,10 +52,6 @@ export const StyledTab = styled.button<StyledTabProps>`
     margin: 0 auto;
   }
 
-  &:focus {
```

Comment:
Previously the parent element was the one focusable and a focus outline was shown, so removing this retains the current behaviour.